js/client/modules/@arangodb/testsuites/export.js

```diff
@@ -209,324 +191,432 @@ function exportTest (options) {
 
-    print(CYAN + Date() + ': Export data (xgmml)' + RESET);
-    args['type'] = 'xgmml';
-    args['graph-name'] = 'UnitTestsExport';
-    testName = "exportXgmml" + idx;
-    results[testName] = pu.executeAndWait(pu.ARANGOEXPORT_BIN, toArgv(args), options, 'arangosh', tmpPath, options.coreCheck);
-    results[testName].failed = results[testName].status ? 0 : 1;
-    
-    testName = "parseXgmml" + idx;
-    try {
-      const filesContent = fs.read(fs.join(tmpPath, 'UnitTestsExport.xgmml'));
-      DOMParser.parseFromString(filesContent);
-      results[testName] = {
-        failed: 0,
-        status: true
-      };
-
-      if (xmlErrors !== null) {
+        results[testName] = {
+          failed: 0,
+          status: true
+        };
+      } catch (e) {
+        results.failed += 1;
         results[testName] = {
           failed: 1,
           status: false,
-          message: xmlErrors
+          message: e
         };
       }
-    } catch (e) {
-      results.failed += 1;
-      results.parseXgmml = {
-        failed: 1,
-        status: false,
-        message: e
-      };
     }
 
+    print(CYAN + Date() + ': Export data (xgmml)' + RESET);
+    {
+      let args = Object.assign({}, commonArgValues);
+      args['type'] = 'xgmml';
+      args['graph-name'] = 'UnitTestsExport';
+      args['collection'] = 'UnitTestsExport';
+      testName = "exportXgmml" + idx;
+      results[testName] = pu.executeAndWait(pu.ARANGOEXPORT_BIN, toArgv(args), options, 'arangosh', tmpPath, options.coreCheck);
+      results[testName].failed = results[testName].status ? 0 : 1;
+    }
+    {
+      testName = "parseXgmml" + idx;
+      try {
+        const filesContent = fs.read(fs.join(tmpPath, 'UnitTestsExport.xgmml'));
+        DOMParser.parseFromString(filesContent);
+        results[testName] = {
+          failed: 0,
+          status: true
+        };
+
+        if (xmlErrors !== null) {
+          results[testName] = {
+            failed: 1,
+            status: false,
+            message: xmlErrors
+          };
+        }
+      } catch (e) {
+        results.failed += 1;
+        results.parseXgmml = {
+          failed: 1,
+          status: false,
+          message: e
+        };
+      }
+    }
     print(CYAN + Date() + ': Export data (xgmml.gz)' + RESET);
-    args['compress-output'] = 'true';
-    testName = "exportXgmmlGz" + idx;
-    results[testName] = pu.executeAndWait(pu.ARANGOEXPORT_BIN, toArgv(args), options, 'arangosh', tmpPath, options.coreCheck);
-    results[testName].failed = results[testName].status ? 0 : 1;
-    
-    testName = "parseXgmmlGz" + idx;
-    try {
-      const filesContent = fs.readGzip(fs.join(tmpPath, 'UnitTestsExport.xgmml.gz'));
-      DOMParser.parseFromString(filesContent);
-      results[testName] = {
-        failed: 0,
-        status: true
-      };
-
-      if (xmlErrors !== null) {
+    {
+      let args = Object.assign({}, commonArgValues);
+      args['compress-output'] = 'true';
```

Comment:
This is now missing
```
 args['type'] = 'xgmml';
```

Reply:
Will address in latest commit with suggestion below of type not being a commong arg for all tests